File: Backend/BackendFinalBossCegep/BackendFinalBossCegep/Model/Compte.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace BackendFinalBossCegep.Model
{
    [Table("utilisateur")]
    public class Compte
    {
        [Column("id")]
        public int Id { get; set; }

        [Column("email")]
        public string Email { get; set; }

        [Column("password")]
        public string Password { get; set; }

        [Column("name")]
        public string Name { get; set; }

        [Column("address")]
        public string Address { get; set; }

        [Column("phone")]
        public string Phone { get; set; }

        [Column("points")]
        public int Points { get; set; }

        public Compte(string email, string password, string name, string address, string phone) 
        {
            this.Email = email;
            this.Password = password;
            this.Name = name;
            this.Address = address;
            this.Phone = phone;
            this.Points = 0;
        }
    }
}

File: Mobile/app/src/main/java/com/example/myapplication/Compte.java
package com.example.myapplication;

import com.google.gson.annotations.SerializedName;

public class Compte
{
    @SerializedName("Id")
    public int id;
    @SerializedName("Email")
    public String email;
    @SerializedName("Password")
    public String password;
    @SerializedName("Name")
    public String name;
    @SerializedName("Address")
    public String address;
    @SerializedName("Phone")
    public String phone;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getAddress() {
        return address;
    }

    public void setAddress(String address) {
        this.address = address;
    }

    public String getPhone() {
        return phone;
    }

    public void setPhone(String phone) {
        this.phone = phone;
    }
}
